packages/flutter/test/material/back_button_test.dart

```diff
@@ -59,4 +60,35 @@ void main() {
     final Icon androidIcon = tester.widget(find.descendant(of: find.byKey(androidKey), matching: find.byType(Icon)));
     expect(iOSIcon == androidIcon, false);
   });
+
+  testWidgets('BackButton semantics', (WidgetTester tester) async {
+    final SemanticsHandle handle = tester.ensureSemantics();
+    await tester.pumpWidget(
+        new MaterialApp(
```

Comment:
nit: add trailing commas and only indent by two spaces.

Reply:
Fixed in https://github.com/flutter/flutter/pull/19046/files